zosia_site/bootstrap/main.tf

```diff
@@ -0,0 +1,19 @@
+provider "google" {
+  project = var.project_id
+  region  = var.tfstate_bucket_region
+}
+
+resource "random_id" "tfstate_bucket_prefix" {
+  byte_length = 8
+}
+
+resource "google_storage_bucket" "tfstate_bucket" {
+  name          = "${random_id.tfstate_bucket_prefix.hex}-tfstate-bucket"
```

Comment:
The idea of the random prefix is ok, but I don’t think it is needed in this case. And by removing it, you will be able to provide a name of the bucket in backend configuration. 

Please keep in mind that in a single bucket you can have multiple states (for different environments or projects). In the future, it might be a good idea to even move the configuration of this bucket outside of this project.

Reply:
According to [docs](https://cloud.google.com/storage/docs/buckets#considerations) all bucket names across all GCP are publicly visible, so they need to be globally unique, that's why I added the random prefix.

I agree, if in the future another project uses terraform, then we should move the bucket config outside of this repo. For now, let's leave it here, not to overcomplicate things unnecessarily.